module/solutions/helper.js

```diff
@@ -121,6 +158,9 @@ module.exports = class SolutionsHelper {
 		programData.createdBy = createdBy
 		programData.startDate = startDate
 		programData.endDate = endDate
+		programData.language = language
+		programData.conversation = conversation
```

Comment:
@Prajwal17Tunerlabs conversation please comment it for now

Reply:
commented conversation as per request.